superset/utils/logging_configurator.py

```diff
@@ -41,19 +41,11 @@ def configure_logging(
         if app_config["SILENCE_FAB"]:
             logging.getLogger("flask_appbuilder").setLevel(logging.ERROR)
 
-        # configure superset app logger
-        superset_logger = logging.getLogger("superset")
-        if debug_mode:
-            superset_logger.setLevel(logging.DEBUG)
-        else:
-            # In production mode, add log handler to sys.stderr.
-            superset_logger.addHandler(logging.StreamHandler())
```

Comment:
I'm unclear on what this does ^^^, but if it does anything that's not the default behavior, removing this line could affect people's production env and is a breaking change

Reply:
See the PR description under ["Cause of duplicate logging"](https://github.com/apache/superset/pull/27777) - by default `logging.basicConfig` triggers this, so this line causes logging duplication.